src/program/Scope.js

```diff
@@ -202,4 +202,4 @@ function getAlias ( chars, i ) {
 	} while ( i > 0 );
 
 	return alias;
-}
\ No newline at end of file
+}
```

Comment:
Thought this newline was caused by an `.editorconfig`, but in fact this is the GitHub online editor I used.